src/DrevOps/Sniffs/TestingPractices/DataProviderPrefixSniff.php

```
@@ -0,0 +1,292 @@
+    // Search for @dataProvider annotations in doc comments.
+    for ($i = 0; $i < $phpcsFile->numTokens; $i++) {
+      if ($tokens[$i]['code'] !== T_DOC_COMMENT_TAG) {
+        continue;
+      }
+
+      if ($tokens[$i]['content'] !== '@dataProvider') {
+        continue;
+      }
+
+      // Find the method name after the tag.
+      $string_ptr = $phpcsFile->findNext(T_DOC_COMMENT_STRING, $i + 1, $i + 3);
```

PHPMD (2.15.0)

The variable $string_ptr is not named in camelCase. (undefined)

(CamelCaseVariableName)